### ISSUE TITLE
Logging details (request-id in particular)

### ISSUE THREAD

**Issue by apneadiving** [googleads/google-ads-ruby#319]
Hi,

I was having a discussion with the support and they did request logs. My logger being setup I did send things like:
```
I, [2021-06-08T10:51:51.619033 #36]  INFO -- : CID: 9455808229, Host: googleads.googleapis.com:443, Method: /google.ads.googleads.v6.services.CampaignService/MutateCampaigns, IsFault: no
D, [2021-06-08T10:51:51.619619 #36] DEBUG -- : Outgoing request: Headers: {"developer-token":"REDACTED","login-customer-id":"7083938566","x-goog-api-client":"gl-ruby/2.6.6 gax/0.3.4 gapic/10.0.0 grpc/1.38.0","x-goog-request-params":"customer_id=9455808229"} Payload: {"customerId":"9455808229","operations":[{"create":{"name":"campaign 1","status":"ENABLED","advertisingChannelType":"SEARCH","networkSettings":{"targetContentNetwork":true,"targetPartnerSearchNetwork":false},"geoTargetTypeSetting":{"positiveGeoTargetType":"PRESENCE_OR_INTEREST","negativeGeoTargetType":"PRESENCE_OR_INTEREST"},"campaignBudget":"customers/9455808229/campaignBudgets/8870575377","finalUrlSuffix":"test-me","biddingStrategy":"customers/9455808229/biddingStrategies/7809070522"}},{"create":{"name":"campaign 2","status":"ENABLED","advertisingChannelType":"SEARCH","networkSettings":{"targetContentNetwork":true,"targetPartnerSearchNetwork":false},"geoTargetTypeSetting":{"positiveGeoTargetType":"PRESENCE_OR_INTEREST","negativeGeoTargetType":"PRESENCE_OR_INTEREST"},"campaignBudget":"customers/9455808229/campaignBudgets/8870575377","finalUrlSuffix":"test-me-twice","manualCpc":{"enhancedCpcEnabled":true}}}],"partialFailure":true}
D, [2021-06-08T10:51:51.619872 #36] DEBUG -- : Incoming response: Payload: {"results":[{"resourceName":"customers/9455808229/campaigns/13436503174"},{"resourceName":"customers/9455808229/campaigns/13436503177"}]}
```

Yet they seem to need the `request-id`. I explained I followed the doc (or so I think) to provide logs (otherwise I would not have any) and I do not have more information. 
They told me to come to you to have explanations and get the `request-id` (I put below the message they sent).

Could you please guide me there?

Thanks! :)

Benjamin

---

The exact reply from the support:
```
You may refer to the links below to see what API response logs and request ID look like. It should be generated along with the API request if the logging of the client library is enabled on your end.
https://developers.google.com/google-ads/api/docs/concepts/field-service#response
https://developers.google.com/google-ads/api/docs/concepts/call-structure#request-id

If you haven't enabled the logging of the client library that you are using, then as I suggested before, please do so by following the guide below for ruby:
https://developers.google.com/google-ads/api/docs/client-libs/ruby/logging

Just in case that there is an issue with configuring it, then I would suggest to reach out to the author of this client library via this link as they are more equipped on this topic.
```

**Comment by mcloonan** [comment]
This was an oversight on our part when we created the logging. It should already be fixed in #302 but that wasn't part of the most recent release, so it will be included in the next released lib version.

That said, the response you provided should be enough for them to help troubleshoot (I work directly with that team also, so please tell them to reach out to me directly if they object to this), and you do not need to specify a request ID for them to assist given that you have provided the full request/response in your message. Sorry for the inconvenience!

**Comment by apneadiving** [comment]
thanks!
